eth2/utils/ssz_types/Cargo.toml

```diff
@@ -14,6 +14,7 @@ serde_derive = "1.0.102"
 serde_hex = { path = "../serde_hex" }
 eth2_ssz = "0.1.2"
 typenum = "1.11.2"
+arbitrary = { version = "0.4", features = ["derive"], optional = true }
```

Comment:
Should this be behind a feature flag?

Reply:
It's set to `optional = true`. Is that not the same as a feature flag with the feature the name of the crate?